python/kvikio/kvikio/benchmarks/utils.py

```diff
@@ -15,6 +16,12 @@
 import kvikio.cufile_driver
 import kvikio.defaults
 
+has_pynvml = False
+with contextlib.suppress(ImportError):
+    import pynvml
+
+    has_pynvml = True
+
```

Comment:
Consider moving this into `pprint_sys_info()`, as `pprint_sys_info()` isn't performance critical.

Reply:
Had put this here as [top-level `import`s are preferred]( https://peps.python.org/pep-0008/#imports ) and it helps keep the code in `pprint_sys_info` a bit more manageable (given other checks occur there)